EDAM_dev.owl

```diff
@@ -54952,7 +54951,6 @@ ows re-sequencing of complete genomes of any given organism with high resolution
     <owl:Class rdf:about="http://edamontology.org/topic_0780">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3070"/>
         <created_in>beta12orEarlier</created_in>
-        <oboInOwl:hasBroadSynonym>Plant science</oboInOwl:hasBroadSynonym>
```

Comment:
Could also stay `broad` instead of `exact` of Plant biology, not no big deal really.